static/app/views/settings/project/projectOwnership/rulesPanel.tsx

```diff
@@ -182,7 +178,7 @@ const InnerPanelBody = styled(PanelBody)`
 `;
 
 const StyledTextArea = styled(TextareaAutosize)`
-  ${inputStyles};
+  ${p => inputStyles(p)};
```

Comment:
This is definitely a typescript / emotion quirk, but you can work around typescript thinking it's not passing this function theme as a prop like this.